internal/ingress/backend/endpoint.go

```diff
@@ -226,6 +221,19 @@ func IsNodeSuitableAsTrafficProxy(node *corev1.Node) bool {
 	return false
 }
 
+// IsPodSuitableAsIPTarget check whether pod is suitable as a TargetGroup's target
+// (currently tested: are all pod's containers ready?).
+func IsPodSuitableAsIPTarget(pod *corev1.Pod) bool {
+	for _, condition := range pod.Status.Conditions {
+		if condition.Type == api.ContainersReady {
+			if condition.Status == api.ConditionTrue {
```

Comment:
This should be `return condition.Status == api.ConditionTrue` to quit whenever the `ContainersReady` condition was found.